client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -17,76 +17,73 @@ import SectionHeader from 'components/section-header';
 // TODO: update supportUrls and maybe create similar mapping for warnings
 function getHoldMessages( translate ) {
 	return {
-		PLACEHOLDER: {
-			title: '',
-			description: '',
-			supportUrl: '',
-		},
 		TRANSFER_ALREADY_EXISTS: {
 			title: translate( 'Installation in progress' ),
-			description: translate( 'Another installation is already in progress.' ),
-			supportUrl: 'https://wordpress.com/help'
+			description: translate( 'Please wait for the other installation to complete, then try again.' ),
+			supportUrl: 'https://wordpress.com/help',
 		},
+		// Handled as a banner
 		NO_BUSINESS_PLAN: {
 			title: translate( 'Business plan required' ),
 			description: translate( 'This feature is only allowed on sites with a business plan.' ),
-			supportUrl: 'https://support.wordpress.com/'
+			supportUrl: 'https://support.wordpress.com/',
```

Comment:
Does this mean we link to the support website when the user clicks on the upgrade banner?